internal/controller/flow_controller.go

```diff
@@ -329,7 +329,7 @@ func (r *FlowReconciler) getConfig(ctx context.Context) (*config.Config, error)
 		return nil, err
 	}
 
-	cfg, err := config.ParseConfig(cfgMap.Data["config"])
+	cfg, err := config.ParseConfig(cfgMap.Data[config.ConfigMapKey])
```

Comment:
if you already updating that can you please remove the const that i used for the configmap and use the same variables and flags that you used for the manager? 
Not a must for this PR, i can do that later.

Reply:
I will align in follow-up